client/core/trade.go

```diff
@@ -674,16 +677,6 @@ func (t *trackedTrade) redeemMatches(matches []*matchTracker) error {
 		)
 	}
 
-	coinToStr := func(assetID uint32, coinID []byte) string {
-		coinStr, err := asset.DecodeCoinID(assetID, coinID)
-		if err != nil {
-			// Coin IDs come from wallet.Redeem so this shouldn't happen. Warn.
-			log.Warnf("redeemMatches: invalid redeemed coin ID %v: %v", coinID, err)
```

Comment:
Noting that by using a package function, we do lose the function name in the warning string.

Reply:
That's why I duped the code, but it's OK.